checklist_plus/text_generation/masked_lm.py

```
@@ -1,335 +1,287 @@
+                    scores[x] = -999999
+        else:
+            for x in scores:
+                scores[x] = scores[x] / len(texts)
+        scores = sorted(scores.items(), key=lambda x:x[1], reverse=True)
+        # print([(list(x[0]), examples[x[0]], x[1]) for x in scores])
+        return [(list(x[0]), examples[x[0]], x[1]) for x in scores]
+
+    def unmask(self, text_with_mask, beam_size=10, candidates=None):
+        if self.url is not None:
+            params = {'text': text_with_mask, 'beam_size': beam_size, 'candidates': candidates}
+            r = requests.post(url='%s/unmask' % self.url, data={'params': json.dumps(params)})
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)